documentation/src/main/wiki/GettingStarted.md

```diff
@@ -260,3 +264,32 @@ This section configures the Services mock used by the example application. It co
 
 Please mind that this an example that depicts a valid setup of the Sample monolith application and it is not fit for use in production environments.
 To learn how to configure Knot.x for use in production, see the [Production](#configuration-1) section.
+
+## Deployment options
+To deploy verticle with advanced options use following properties:
+```json
+{
+    ... 
+  "mockService": {
+    "config": {
+      "mock.data.root": "mock/service",
+      "http.port": 3000
+    },
+    "worker" : false,
+    "multiThreaded": false,
+    "isolationGroup": "null",
+    "ha": false,
+    "extraClasspath": [],
+    "instances": 2,
+    "isolatedClasses": []
```

Comment:
Do we need all other properties form deployment options defined? Moste of them are default values. I think it would be better to give user a reference to available [vert.x deployment options](http://vertx.io/docs/apidocs/io/vertx/core/DeploymentOptions.html).